src/rust/ensogl/src/display/shape/text/text_field/cursor.rs

```diff
@@ -635,7 +646,7 @@ mod test {
         let properties     = TextFieldProperties::default(&mut fonts);
         let mut content    = TextFieldContent::new(text,&properties);
         let mut navigation = CursorNavigation
-            {selecting:true, ..CursorNavigation::default(&mut content)};
+            {selecting:true, ..CursorNavigation::new(&mut content)};
```

Comment:
And I still don't see the test cases for PageUp/PageDown navigation. I suspect that you missed it because I wrote it on "overall review comment", so now I put it here :)

Reply:
I got a test for it :)